packages/gamut/src/GridForm/GridFormInputGroup/index.tsx

```diff
@@ -120,7 +121,11 @@ export const GridFormInputGroup: React.FC<GridFormInputGroupProps> = ({
         );
     }
   };
-  if (field.type === 'hidden' || field.type === 'sweet-container')
+  if (
+    field.type === 'hidden' ||
+    field.type === 'sweet-container' ||
+    field.type === 'customGroup'
+  )
```

Comment:
⛳   + May help to have the explicit `{}` block here :)
```suggestion
  if (['hidden', 'sweet-container', 'custom-group'].includes(field.type)) {
    return getInput()
  }
  ```